pkg/metadata/annotations/annotations.go

```diff
@@ -1,6 +1,6 @@
 package annotations
 
-// skip reconcile.
+// ManagedByODHOperator is used to denote if a resource/component should be reconciled - when true, reconcile.
```

Comment:
![hey-you!](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExNjJpMWx5bmJ3eDRtYW5hZzltbzlwYzkweWZ3a3JvM2NsNHZxeTVoYyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/Q6qk4WvxyiezknWJEz/giphy-downsized.gif)

Nice little tidy! Thanks!